src/controllers/wishlist.controller.js

```diff
@@ -0,0 +1,56 @@
+const Wishlists = require("../models/wishlist.model");
+const asyncHandler = require("../utils/asyncHandler.util");
+
+const getWishlist = asyncHandler(async (req, res) => {
+    const wishlist = await Wishlists.findOne({ user: req.user._id }).populate('products');
+    res.json(wishlist?.products || []);
+});
```

Comment:
take out `wishlist?.products || []` logic out, for readability
` e.g. wishlistItems = wishlist?.products || []`

I have one suggestion, always use object to send as a response json, so that you scale it later.